js/script.js

```diff
@@ -210,4 +210,26 @@ function controlMobileMenu() {
     });
   });
 }
+
 controlMobileMenu();
+
+// email form validation
+/*  form id="get-in-touch"
+    id="user-email"
+    submit button class = "btn-get"
+    error message class="form-email-error-message" */
+
+const emailInput = document.querySelector('#user-email');
+const submitButton = document.querySelector('.btn-get');
+const emailErrorMessage = document.querySelector('.form-email-error-message');
+function validationError(e) {
+  if ((emailInput.value).match(/[A-Z]/)) {
+    emailErrorMessage.style.display = 'block';
+    submitButton.type = 'button';
+    e.preventDefault();
+  } else {
+    emailErrorMessage.style.display = 'none';
+    submitButton.type = 'submit';
+  }
+}
+submitButton.addEventListener('click', validationError);
```

Comment:
- [ **OPTIONAL** ] Good :+1: job with your email validation as it works as expected :100:. However, I noticed that the error message shows up even if before filling any input field(i.e. just when a user launches your app.). I suggest you update your code in the way the error message only shows up when the user put `email` with uppercase characters.